alan-hadyk/grocify
# Migrate from Rust to Go

**Note: Start with `README` in `/server-new` folder and try to run the server**

# Testing scenario

1. Run the server and open http://localhost:3000/

2. Run the mutation:

```
mutation createUser {
  createUser(input:{
    password:"12345678",
    email:"john@gmail.com",
    preferredLanguage:Pl
  }){
    id
    email
    createdAt
    preferredLanguage
  }
}
```

This should return the created user

3.  Run the same mutation again. This should throw an error:

```
{
  "errors": [
    {
      "message": "user with email john@gmail.com already exists",
      "path": [
        "createUser"
      ]
    }
  ],
  "data": null
}
```
